### PR TITLE
Update PostgreSQL version.

### PR DESCRIPTION
**Pull request by nepella**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 variable "alt_hostname" {
   description = "Specify an alternative hostname for the public website url (instead of public_zone_name)"
-  type    = map(object({
+  type    = map(object({https://github.com/avalonmediasystem/avalon-terraform
```
Comment (cjcolvar) [inline variables.tf:3]: I don't understand why the repo url is added here.  Is this a typo or does it serve some purpose?

File: variables.tf [hunk 2]
```diff
@@ -165,7 +165,7 @@ variable "hosted_zone_name" {
 }
 
 variable "postgres_version" {
-  default = "14.4"
+  default = "14.10"
```
Comment (cjcolvar) [inline variables.tf:168]: :+1:

File: variables.tf (hunk 1 above)
Comment (nepella) [reply variables.tf]: @cjcolvar: That's definitely a typo, likely an accidental middle-button paste. I'll update my pull request.

**Review by cjcolvar — APPROVED** [review]
Thanks for updating this!

**Pull request merged**